book/07-git-tools/sections/replace.asc

```diff
@@ -188,9 +188,9 @@ committer Scott Chacon <schacon@gmail.com> 1268712581 -0700
 fourth commit
 ----
 
-Remember that the actual parent of `81a708d` was our placeholder commit (`622e88e`), not `9c68fdce` as it states here.
+Tandaan na ang aktwal na magulang sa `81a708d` ay ang aming na commit (`622e88e`), hindi `9c68fdce` gaya ng sinasabi nito dito.
```

Comment:
`aming na commit` to `aming placeholder na commit`

Reply:
done change